# deacon_rockfish_RADseq
Commands and input files for population genetic analysis of Deacon Rockfish using RADseq. Used for: Vaux F, Rasmuson LK, Kautzi LA, Rankin PS, Blume MTO, Lawrence KA, Bohn S, Oâ€™Malley KG. (2019). Sex matters: otolith shape and genomic variation in deacon rockfish (Sebastes diaconus). Ecology and Evolution 9, 13153-13173. https://doi.org/10.1002/ece3.5763

# deacon_rockfish_STACKS_and_filtering
Lists input commands used for the STACKS analysis and subsequent analyses used for filtering in VCFtools, plink, and paralog-finder (HDplot), as well as BWA.

# Other files on GitHub
These are input files (e.g. population maps, blacklists) used for STACKS and the other tools.

# Genetic sampling, and genotype and loci fasta files
Genetic sampling, and genetic data (vcf, genepop, and structure genotype files, and the fasta consensus file for the 92 outlier loci) are available from Dryad: https://doi.org/10.5061/dryad.8kprr4xj0

# Demultiplexed reads
Demultiplexed forward and reverse DNA sequence reads for the deacon rockfish sequenced in this study are openly available on the NCBI sequence read archive (SRA) under: PRJNA560239, http://www.ncbi.nlm.nih.gov/bioproject/560239

# Raw Illumina reads
Available on request from Kathleen O'Malley (State Fisheries Genomics Lab, Oregon State University: https://agsci.oregonstate.edu/state-fisheries-lab/people)

# Further information
See this page: https://sites.google.com/view/evauxlution/data
